docsets/xarray/docset.json

```diff
@@ -1,16 +1,20 @@
 {
     "name": "xarray",
-    "version": "2023.09.0",
+    "version": "2025.06.1",
     "archive": "xarray.tgz",
     "author": {
-        "name": "Florian Knoop, Xavier Yang",
-        "link": "https://github.com/flokno, https://github.com/ivaquero"
+        "name": "Florian Knoop, Xavier Yang, Clément Haëck",
+        "link": "https://github.com/flokno, https://github.com/ivaquero, https://github.com/Descanonge"
     },
     "aliases": [
         "Python",
         "N-D labeled arrays and datasets in Python"
     ],
     "specific_versions": [
+        {
+            "version": "2025.06.1",
+            "archive": "versions/2025.06.1/xarray.tgz"
```

Comment:
Thank you! Please include this file as well the next time you update the docset. I've added it this time, by coping `xarray/xarray.tgz`